src/main/java/uk/gov/justice/digital/delius/data/api/UserDetails.java

```diff
@@ -29,4 +29,6 @@
     private boolean enabled;
     @ApiModelProperty(value = "Roles For this User", position = 6, allowEmptyValue = true)
     private List<UserRole> roles;
+    @ApiModelProperty(value = "The username of the user", example = "test.user", allowEmptyValue = true, position = 7)
```

Comment:
why are you allowing empty values - `allowEmptyValue` is set to true?

Reply:
This object is used by another api call as well that doesn't set the username. Discussed this originally with Andy Marke, but will have a second chat about it. Maybe we can extend to another object.

Reply:
that makes sense then :+1: